src/main/java/com/everest/displaystrategy/DisplayCarStrategy.java

```diff
@@ -0,0 +1,40 @@
+package com.everest.displaystrategy;
+
+import com.everest.display.Display;
+import com.everest.parkinglot.ParkingLot;
+
+import java.util.ArrayList;
+
+public class DisplayCarStrategy implements DisplayStrategy{
+    Display display=new Display();
+
+    @Override
+    public void displayFreeCount(ParkingLot parkingLot) {
+        ArrayList<ArrayList<Integer>> res=display.display(parkingLot, 3, parkingLot.getSlotsInEachFloor());
+        int temp=parkingLot.getNoOfFloors();
+        while(temp>0){
+            System.out.print("No. of free slots for "+"CAR"+" on Floor "+((parkingLot.getNoOfFloors()-temp)+1)+" : "+res.get(parkingLot.getNoOfFloors()-temp).size()+"\n");
+            temp--;
+        }
+    }
+
+    @Override
+    public void displayFreeSlots(ParkingLot parkingLot) {
```

Comment:
This is bad idea having same duplicate code for each vehicle. It should be one source file and should work for any case